src/locales/en.json

```
@@ -130,42 +130,43 @@
-    "content": "What content types can I search for using the Meta Search feature?",
-    "content-types": "Meta Search on {openverse} currently supports image, audio, and video search on external sites.",
+    "link": "sources here",
+    "license": "However, there are many sources of CC-licensed and public domain media which we aren’t able to surface results from within the {openverse} interface yet. This might be because they do not offer a public API, or that our contributors have not yet had time to integrate them into Openverse. These are valued sources and we want to make sure that you are able to find the best openly licensed materials possible, regardless of where they are located.",
     "use": " Use ",
     "images": {
-      "title": "How does the Meta Search feature work for indexed media types?",
-      "demo-label": "Layout of Meta Search for indexed media type, like images",
+      "title": "How does the External Sources feature work for indexed media types?",
+      "demo-label": "Layout of External Sources for indexed media type, like images",
       "content": "{openverse} supports built-in search for some media types, such as images. When you enter a query on the {openverse} homepage, a page of results will be displayed. Scrolling to the bottom of the page, you have two choices:",
       "continue": "{load-more}, and continue browsing in {openverse}.",
       "not-finding": "See the section “{not-finding}” below the results.",
       "info": "The “{not-finding}” section lists other sources of CC-licensed and public domain content. Your search query will be preloaded, and if supported, you can select a {use} filter to proactively filter your results. Click on the button for the external source you’d like to search, and your results will open in a new tab."
     },
     "audio-video": {
-      "title": "How does the Meta Search feature work for non-indexed media types?",
-      "demo-label": "Layout of Meta Search for non-indexed media type, like video",
+      "title": "How does the External Sources feature work for non-indexed media types?",
+      "demo-label": "Layout of External Sources for non-indexed media type, like video",
       "content": "{openverse} does not yet support built-in search for some media types, such as videos. For these media types, click on the appropriate tab and you’ll see sources where you can find CC-licensed content.",
-      "filter": "The Filters bar on the left includes the {use} filter, which may be supported by some meta search sources. Apply these before clicking on the button for the external source you’d like to search."
+      "filter": "The Filters bar on the left includes the {use} filter, which may be supported by some external Sources sources. Apply these before clicking on the button for the external source you’d like to search."
     },
     "new": {
-      "title": "Can I suggest new sources for Meta Search?",
+      "title": "Can I suggest new sources for External Sources?",
```

This sentence does not read well. Consider changing to:
```suggestion
      "title": "Can I suggest new external sources?",
```